templates/tab/js/default/public/index.html

```diff
@@ -1,7 +1,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
aHA! not exactly. but good catch!  https://stackoverflow.com/questions/19506028/do-you-need-to-close-meta-and-link-tags-in-html

Reply:
Thanks for comment! As it dose no harm and for consistency, let's remain the slash.